docs/users/extend/in-container-configuration.md

```diff
@@ -1,11 +1,12 @@
 ## In-Container home directory configuration
```

Comment:
Aren't you going to want to improve this title?

Reply:
yeah I considered that but wasn't sure.   I guess it'd depend on whether you think it'd be good to break it out into smaller more specific pages like I mentioned in the other comment.  What do you think?